modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/1_application.json

```diff
@@ -305,6 +305,466 @@
+			"email": "lroskrugh@canalblog.com",
+			"externalReferenceCode": "RAYAPP-41",
+			"firstName": "Loydie",
+			"lastName": "Roskrug",
+			"phone": "702-136-5045",
+			"productCategory": "",
+			"productId": "RAYAP-001",
+			"productName": "Auto",
+			"r_accountToApplications_accountEntryERC": "RAYACCOUNT-002",
+			"state": "NV",
+			"submitDate": "2022-05-11",
+			"user": "Test",
+			"website": "Null",
+			"zip": "89140"
+		},
+		{
+			"address": "86969 Vahlen Way",
+			"addressApt": "3",
+			"applicationCreateDate": "2022-04-14",
+			"applicationStatus": "underwriting",
+			"businessCategoryId": "",
+			"city": "Fresno",
+			"dataJSON": "{\"driverInfo\":{\"form\":{\"ageFirstLicensed\":16,\"hasAccidentOrCitations\":\"yes\",\"accidentCitation\":\"Accident - At fault\",\"firstName\":\"Jolie\",\"gender\":\"Female\",\"lastName\":\"Zecchi\",\"maritalStatus\":\"Widowed\",\"millitaryAffiliation\":null,\"occupation\":\"Other\",\"governmentAffliation\":\"State Firefighters Association\",\"otherOccupation\":\"Account Coordinator\",\"relationToContact\":\"Self\",\"highestEducation\":\"Phd\",\"creditRating\":\"Very Good\"}},\"vehicleInfo\":{\"form\":{\"annualMileage\":3564,\"year\":2018,\"make\":\"Toyota\",\"model\":\"Corolla LE\",\"features\":\"Anti-lock braking system that is factory-installed,Lane-Departure Warning & Lane-Keeping Assist,Forward-Collision Warning,Blind-Spot Monitoring\",\"ownership\":\"lease\",\"primaryusage\":\"commuting\"}},\"coverage\":{\"form\":{\"bodilyInjury\":\"$100,000 / $ 300,000\",\"medical\":\"$5,000\",\"propertyDamage\":\"$300,000\",\"uninsuredOrUnderinsuredMBI\":\"$300,000\",\"uninsuredOrUnderinsuredMPD\":\"$30,000 / $60,000\"},\"vehicles\":{\"collision\":\"$250\",\"comprehensive\":\"$500\"}}}",
+			"email": "jzecchi7@google.cn",
+			"externalReferenceCode": "RAYAPP-42",
+			"firstName": "Jolie",
+			"lastName": "Zecchi",
+			"phone": "559-835-6042",
+			"productCategory": "",
+			"productId": "RAYAP-001",
+			"productName": "Auto",
+			"r_accountToApplications_accountEntryERC": "RAYACCOUNT-003",
+			"state": "CA",
+			"submitDate": "2022-04-14",
+			"user": "",
```

Comment:
Why does this field empty for some and "Test" for others? If it is not useful, please delete it.

Reply:
Ok